### PR TITLE
Add multiprice recipe

### PR DESCRIPTION
**Pull request by GoZOo**

### CODE REVIEW

File: v2/recipes/multiprice.md [hunk 1]
```diff
@@ -88,17 +88,34 @@ use Drupal\commerce_price\Resolver\PriceResolverInterface;
...
+   */
+  protected $languageManager;
+
+  /**
+   * Constructs a new CommerceMulticurrencyResolver object.
+   *
+   * @param \Drupal\Core\Language\LanguageManagerInterface $language_manager
+   *   The language manager.
+   */
+  public function __construct(LanguageManagerInterface $language_manager) {
```
Comment (vasike) [inline v2/recipes/multiprice.md]: Working on https://www.drupal.org/node/2831954#comment-11906059
and using the implementation suggested here, the tests won't work with $language_manager construct argument.

Please check my PR for the issue https://github.com/drupalcommerce/commerce/pull/626
And maybe we could get the right solution for how this CommerceMulticurrencyResolver should be done

**Comment by luksak** [comment]
I think we should document a few different use cases for handling multicurrency. The most common use case I have is the following:

* Initially the currency is detected based on IP and set by zone/country
* The user can switch currencies using a currency switcher

Could we document that behavior as well?

**Comment by bojanz** [comment]
A currency switcher is out of scope, but we can show how the geoip module can be used to get the ip country in the price resolver.

**Comment by mglaman** [comment]
I'm going to close this as it is pretty dated. And there are contrib incoming for this. Anyone is free to re-open a new PR to help cover this in the Developer Guide as "Multiple currency support"

**Pull request closed without merging**